examples/drupal_loadtest.rs

```diff
@@ -164,7 +164,9 @@ async fn drupal_loadtest_login(client: &mut GooseClient) {
 /// Post a comment.
 async fn drupal_loadtest_post_comment(client: &mut GooseClient) {
     let nid: i32 = rand::thread_rng().gen_range(1, 10_000);
-    let response = client.get(format!("/node/{}", &nid).as_str()).await;
+    let node_path = format!("node/{}", &nid);
+    let comment_path = format!("/comment/reply/{}", &nid);
+    let response = client.get(&node_path).await;
```

Comment:
In a slightly more complicated example, we can reduce the added overhead by calculating `node_path` and `comment_path` one time, then using the variable to set errors.

Reply:
To be honest I would just encapsulate in the response, so:

`client.set_failure(response);`

might be wanted to still do it manually so having the `<Option>, string` interface as well might be good, but response should have all data on the request.

Reply:
Ok, I've been hesitant to add a lot of abstraction, but it would clean this up nicely, and work with any client and/or protocol. I like this plan, I will implement in a followup once this PR is merged.

Reply:
Issue created: https://github.com/tag1consulting/goose/issues/40